axelor-account/src/main/java/com/axelor/csv/script/ImportAnalyticGrouping.java

```diff
@@ -0,0 +1,19 @@
+package com.axelor.csv.script;
+
+import com.axelor.apps.account.db.AnalyticGrouping;
+import com.axelor.common.ObjectUtils;
+import com.axelor.exception.AxelorException;
+import java.util.Map;
+
+public class ImportAnalyticGrouping {
+
+  public Object importAnalyticGrouping(Object bean, Map<String, Object> values)
+      throws AxelorException {
+    assert bean instanceof AnalyticGrouping;
+    AnalyticGrouping analyticGrouping = (AnalyticGrouping) bean;
+    if (ObjectUtils.isEmpty(analyticGrouping.getName())) {
+      analyticGrouping.setFullName(analyticGrouping.getCode() + "_" + analyticGrouping.getName());
+    }
```

Comment:
We should have only one way to compute the fullname. Means, we can define a java method an call it from xml view when we update an analyticGrouping and also call it from the xml config file to import. Else, it better to use xml action in xml view and in a bind in import config file.